README.md

```diff
@@ -1,5 +1,5 @@
 # e2e-behaviour-testing
-
+Clear and scalable e2e tests written in English only
```

Comment:
I am not sure that this description is appropriate. @NikAtSpark, @DanAtSpark, do you have any ideas?

Reply:
Maybe just something like - Clear and scalable e2e tests written in plain English

Reply:
I like Nik's description more. @DanAtSpark, please change the description if you have no objections.

Reply:
I like this description more. I've pushed a commit with those changes